Nonprofit-Exchange-Hub/web-app
# web-app#307: new reset password styling

https://github.com/orgs/Nonprofit-Exchange-Hub/projects/14/views/1?pane=issue&itemId=25095839

## Dev Summary

Update the password reset page based on latest mock - [password reset page](https://www.figma.com/file/K7DT5jYcLKRc9epmNrO0N7/UX---Dev-Handoff?node-id=228-5233&t=0a5133SJk6atcWfw-0)


This commit includes:
- new image to display on the password reset page
- added `id`, `name`, `onBlur`, `sublabel` props to `PasswordInput` component
- updated `SetNewPassword` component based on mocks

## Test Plan


repro steps:

1. go to http://localhost:3000/set-new-password 
2. verify that the page reflects the latest mocks
3. check if validation is applied when setting new password (error message is displayed and submit button disabled):
- if password is less than minimum # of characters
- if there's a mismatch of passwords entered



<img width="1438" alt="Screenshot 2023-04-24 at 11 57 18 PM" src="https://user-images.githubusercontent.com/2562488/234198265-8c268e6d-6e8b-4962-846f-1102f40ea5cd.png">



## Resources

- [stackoverflow is cool](stackoverflow.com)
